index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf8" />
+<html>
```

Comment:
Let's  please rename this failure-stats.html or something more specific than "index". Also, how about keeping render.js and style.css inline? This is simple enough that a single file is easier to handle IMHO. If you want to keep them separate, please rename them.